spec/controllers/spotlight/bulk_actions_controller_spec.rb

```diff
@@ -2,12 +2,18 @@
 
 describe Spotlight::BulkActionsController, type: :controller do
   routes { Spotlight::Engine.routes }
-  let(:exhibit) { FactoryBot.create(:exhibit) }
+  let(:exhibit) { FactoryBot.create(:skinny_exhibit) }
 
-  before do
-    allow(Spotlight::ChangeVisibilityJob).to receive(:perform_later)
+  # rubocop:disable RSpec/BeforeAfterAll
+  before(:all) do
+    ActiveJob::Base.queue_adapter = :test
```

Comment:
fwiw, i couldn't find any reason we're using `:inline` at all: https://github.com/projectblacklight/spotlight/pull/2698

Reply:
Got it thanks.. I'll cherry pick that in and then remove the before/after blocks.

Reply:
Scratch that.. I still have to set `ActiveJob::Base.queue_adapter = :test` as the ` Internal::Application.config.active_job.queue_adapter` change doesn't seem to have the same impact.